### PR TITLE
ACU Upgrades for Gari

### PR DESCRIPTION
**Pull request by speed2CZ**
Even noob would have some upgrades on ACU at that point.

### CODE REVIEW

File: X1CA_Coop_001/X1CA_Coop_001_script.lua [hunk 1]
```diff
@@ -1504,6 +1504,9 @@ function IntroMission3()
             -- Order CDR
             ScenarioInfo.OrderCDR = ScenarioUtils.CreateArmyUnit('Order', 'Order_ACU')
             ScenarioInfo.OrderCDR:SetCustomName(LOC '{i Gari}')
+            ScenarioInfo.OrderCDR:CreateEnhancement('ResourceAllocationAdvanced')
+            ScenarioInfo.OrderCDR:CreateEnhancement('EnhancedSensors')
+            ScenarioInfo.OrderCDR:CreateEnhancement('T3Engineering')
```
Comment (ChrisKitching) [inline X1CA_Coop_001/X1CA_Coop_001_script.lua]: So these enhancements show up right without you explicitly showing the bones - why did you need to show the bones  by hand in the other PR?

Comment (speed2CZ) [inline X1CA_Coop_001/X1CA_Coop_001_script.lua]: because there is no PlayCommanderWarpInEffect() that would hide them. If you use just CreateArmyUnit then it will create ACU with all upgrade bones

Comment (ChrisKitching) [inline X1CA_Coop_001/X1CA_Coop_001_script.lua]: Ew.

**Comment by ChrisKitching** [comment]
Merged.
